### PR TITLE
[SD-4033] use translate to rename Urgency to News Value

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: app/scripts/translations.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
 /* jshint -W100 */
 	gettextCatalog.setStrings('de', {"My Profile":"DE - My Profile"});
+    gettextCatalog.setStrings('en', {
+        "URGENCY": "News Value",
+        "Urgency": "News Value",
+        "Urgency stats": "News Value stats",
+        "urgency": "News Value"
```
Comment (ahilles107) [inline app/scripts/translations.js]: Why we have so many (3) keys (with just different size of letters) keys for this same translation string? Translators will need translate  "News Value" at least 3 times that string.

Comment (petrjasek) [inline app/scripts/translations.js]: because it's case sensitive

Comment (ahilles107) [inline app/scripts/translations.js]: I see, but value isn't - so we will have 3 this same translations for different keys. Which is bad.

**Comment by marwoodandrew** [comment]
@petrjasek The thing I don't get is why gettext would not translate the value in seach.js for the sort order, is there something else I need to do there?

File: app/scripts/translations.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
 	gettextCatalog.setStrings('de', {"My Profile":"DE - My Profile"});
+    gettextCatalog.setStrings('en', {
+        "URGENCY": "NEWS VALUE",
+        "Urgency": "News Value",
+        "urgency": "news value",
+        "Urgency stats": "News Value stats"
+    });
 /* jshint +W100 */
-}]);
+}]);
```
Comment (ahilles107) [inline app/scripts/translations.js]: missing empty line at end of file

**Comment by ioanpocol** [comment]
please rebase

**Comment by petrjasek** [comment]
:+1:

File: app/scripts/translations.js (hunk 1 above)
Comment (petrjasek) [inline app/scripts/translations.js]: right, good point

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**